src/com/example/powerup/SessionHistory.java

```diff
@@ -0,0 +1,6 @@
+package com.example.powerup;
+
+public class SessionHistory {
+	public static Integer CurrQID = 1;
+	public static Integer CurrSessionID = 1;
```

Comment:
Are these variables going to be updated anytime? If so, they should not be static. Otherwise, they should be in uppercase (CURR_QID, CURR_SESSION_ID). If they will be updated, make them private, non-static and lowercase